spec/system/addresses_spec.rb

```diff
@@ -3,7 +3,7 @@
 RSpec.describe 'Addresses', type: :system do
   it '既存の住所が登録済みの時、newページに遷移せずshowにリダイレクトされること' do
     user = create(:user)
-    create(:address, user:)
+    create(:address, :with_user, user:)
```

Comment:
user指定してるんだから、with_user不要じゃない？
```suggestion
    create(:address, user:)
```